socs/agents/srs_cg635m/drivers.py

```diff
@@ -0,0 +1,129 @@
+# This device uses the Prologix GPIB interface
+from socs.common.prologix_interface import PrologixInterface
+
+
+class SRS_CG635m_Interface(PrologixInterface):
```

Comment:
By convention (and following PEP08) we use CapWords for classnames, so `SRSCG635mInterface`.